[2023-10-02T03:43:04.549+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_LC8.get_data scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:43:04.664+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: Valdi_LC8.get_data scheduled__2023-10-01T00:00:00+00:00 [queued]>
[2023-10-02T03:43:04.670+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:43:04.673+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 2
[2023-10-02T03:43:04.686+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2023-10-02T03:43:04.772+0000] {taskinstance.py:1389} INFO - Executing <Task(PythonOperator): get_data> on 2023-10-01 00:00:00+00:00
[2023-10-02T03:43:04.797+0000] {standard_task_runner.py:52} INFO - Started process 226 to run task
[2023-10-02T03:43:04.850+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Valdi_LC8', 'get_data', 'scheduled__2023-10-01T00:00:00+00:00', '--job-id', '17', '--raw', '--subdir', 'DAGS_FOLDER/P2G8_Rivaldi_Valensia_DAG.py', '--cfg-path', '/tmp/tmpw1ap_g6r', '--error-file', '/tmp/tmpqf1df2b7']
[2023-10-02T03:43:04.863+0000] {standard_task_runner.py:80} INFO - Job 17: Subtask get_data
[2023-10-02T03:43:05.551+0000] {task_command.py:371} INFO - Running <TaskInstance: Valdi_LC8.get_data scheduled__2023-10-01T00:00:00+00:00 [running]> on host 5f5d0b98abe4
[2023-10-02T03:43:06.003+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Valdi
AIRFLOW_CTX_DAG_ID=Valdi_LC8
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2023-10-01T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-10-01T00:00:00+00:00
[2023-10-02T03:43:06.424+0000] {python.py:173} INFO - Done. Returned value was: None
[2023-10-02T03:43:06.563+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=Valdi_LC8, task_id=get_data, execution_date=20231001T000000, start_date=20231002T034304, end_date=20231002T034306
[2023-10-02T03:43:06.643+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-10-02T03:43:06.828+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
